ext/js/language/text-scanner.js

```diff
@@ -1382,13 +1382,13 @@ export class TextScanner extends EventDispatcher {
                 return input.scanOnPenRelease;
         }
         switch (this._penPointerState) {
-            case 1: // hovering
+            case 1:
                 return input.scanOnPenHover;
-            case 2: // touching
+            case 2:
                 return input.scanOnPenMove;
-            case 3: // hovering after touching
+            case 3:
                 return input.scanOnPenReleaseHover;
-            default: // not active
+            case 0:
```

Comment:
Q: Will ts error for an incomplete pattern match if something needs to be gets added to the enum? That would be a nice benefit

Reply:
Yes, because in that case the switch statement returns undefined, and that is incompatible with the return type provided you typed the return type correctly.

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.md